client.go

```diff
@@ -561,6 +561,12 @@ type EntityErrors struct {
 	warns []error
 }
 
+// NewEntityErrors returns a wrapper for errors encountered while parsing
+// entity struct tags.
+func NewEntityErrors(warns []error) *EntityErrors {
+	return &EntityErrors{warns: warns}
```

Comment:
In the new registry, because it's searching for entities rather than being given valid entities, I wanted to re-use the existing `dosa.EntityErrors` type but because `warns` is unexported, I need a constructor to call from this new package.